docs/pages/_document.js

```diff
@@ -2,13 +2,15 @@ import * as React from 'react';
 import { ServerStyleSheets } from '@material-ui/styles';
 import { ServerStyleSheet } from 'styled-components';
 import createEmotionServer from '@emotion/server/create-instance';
+import { CacheProvider } from '@emotion/react';
 import Document, { Html, Head, Main, NextScript } from 'next/document';
 import { LANGUAGES_SSR } from 'docs/src/modules/constants';
 import { pathnameToLanguage } from 'docs/src/modules/utils/helpers';
 import { themeColor } from 'docs/src/modules/components/ThemeContext';
-import { cacheLtr } from 'docs/pages/_app';
+import createCache from '@emotion/cache';
 
-const { extractCritical } = createEmotionServer(cacheLtr);
+const cache = createCache({ key: 'css', prepend: true });
```

Comment:
Hm, this doesn't look like creating the cache per render - unless in your setup this file is reexecuted each render. Don't you have global styles leaking between pages right now?

Reply:
Good catch! I reverted back the `getCache` method. Thanks :)